agent/app/agent.go

```diff
@@ -866,6 +864,13 @@ func (agent *ecsAgent) startAsyncRoutines(
 	// Start sending events to the backend
 	go eventhandler.HandleEngineEvents(agent.ctx, taskEngine, client, taskHandler, attachmentEventHandler)
 
+	err := statsEngine.MustInit(agent.ctx, taskEngine, agent.cfg.Cluster, agent.containerInstanceARN)
+	if err != nil {
+		seelog.Warnf("Error initializing metrics engine: %v", err)
+		return
+	}
+	go statsEngine.StartMetricsPublish()
+
 	telemetrySessionParams := tcshandler.TelemetrySessionParams{
```

Comment:
Sorry I missed it before, but why are we still using the old handler instead of the new shared one in ecs-agent.

Reply:
Oops! Looks like there is a PR open yet to be merged. Will wait for it.